### DISCUSSION TITLE
creating a para_dim=2 spline with two para_dim=1 splines

### DISCUSSION THREAD

**Discussion by j042** [tataratat/splinepy#264] [General]
I roughly think of something like this:
```
# from
*-----*-----*--*---*----* 



*-*--*-----*------*-----*
```

```
# to
*-*--*--*---*--*--*--*-*----* 
|                           |
|                           |
|                           |
*-*--*--*---*--*--*--*-*----*
```

**Comment by mkofler96** [comment]
This would be a reall cool feature. Many CAD applications use this a start for freeform surfaces. Usually its also possible to use multiple 1D param splines:
![image](https://github.com/tataratat/splinepy/assets/18218171/0d28103a-bc9a-4bc4-8816-d8848b19152c)
from https://en.m.wikipedia.org/wiki/Freeform_surface_modelling

Btw. a thickening option (perpendicular to the 2D surface) would then also be cool to create the volume 🤔
